cmd/dra-example-kubeletplugin/cdi.go

```diff
@@ -20,13 +20,15 @@ import (
 	"fmt"
 	"os"
 
+	"sigs.k8s.io/dra-example-driver/pkg/consts"
+
 	cdiapi "tags.cncf.io/container-device-interface/pkg/cdi"
 	cdiparser "tags.cncf.io/container-device-interface/pkg/parser"
 	cdispec "tags.cncf.io/container-device-interface/specs-go"
 )
 
 const (
-	cdiVendor = "k8s." + DriverName
+	cdiVendor = "k8s." + consts.DriverName
```

Comment:
I'm not generally a fan of `consts` packages but that seemed at least better than duplicating the variable. Open to suggestions for a better name or a place for this variable where it can be used by two different `main` packages. This change is in a separate commit from the rest of the webhook changes in case that makes this easier to review at all, but please let me know if I should squash.

Reply:
Why not, for what it's worth, I keep a version in a package, it can be overriden during build-time so we don't have to change the version manually - commit hash, for example, but the release version too.